src/Pin/index.js

```diff
@@ -73,32 +77,32 @@ export default {
     )
     pinDialog = null
   },
-  submit(pin) {
+  submit(pin, context = contextItems[0]) {
```

Comment:
It would be nice if we can advise the developer that it's recommended to pass an explicit `context`. This means that we should remove the default value here, and assign it inside the function if left empty (and add a `Log.info()`)